plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestingLakekeeperCatalog.java

```diff
@@ -0,0 +1,214 @@
+        postgresqlContainer.withNetwork(net).start();
+
+        String pgConnectionString = "postgresql://" + postgresqlContainer.getUsername() + ":" + postgresqlContainer.getPassword() + "@" + postgresqlContainer.getContainerName().substring(1) + ":" + PostgreSQLContainer.POSTGRESQL_PORT + "/" + postgresqlContainer.getDatabaseName();
+
+        migrator = new GenericContainer<>(lakekeeperCatalogImage);
+        migrator.withEnv(lakekeeperPGEncryptionKey, pgEncryptionKey)
+                .withEnv(lakekeeperDbUrlReadKey, pgConnectionString)
+                .withEnv(lakekeeperDbUrlWriteKey, pgConnectionString).withNetwork(postgresqlContainer.getNetwork())
+                .withCommand("migrate").start();
+
+        minio = new GenericContainer<>("bitnami/minio:latest");
+        minio.withEnv(Map.of("MINIO_ROOT_USER", MINIO_ROOT_USER,
+                        "MINIO_ROOT_PASSWORD", MINIO_ROOT_PASSWORD,
+                        "MINIO_API_PORT_NUMBER", "%d".formatted(MINIO_PORT),
+                        "MINIO_CONSOLE_PORT_NUMBER", "9001",
+                        "MINIO_SCHEME", "http",
+                        "MINIO_DEFAULT_BUCKETS", DEFAULT_BUCKET))
+                .withNetwork(postgresqlContainer.getNetwork())
+                .withNetworkAliases(MINIO_ALIAS)
+                .waitingFor(Wait.forSuccessfulCommand("mc ls local | grep examples"))
+                .withExposedPorts(MINIO_PORT, 9001).start();
+
+        lakekeeperCatalog = new GenericContainer<>(lakekeeperCatalogImage);
+        lakekeeperCatalog.withEnv(Map.of(lakekeeperPGEncryptionKey, pgEncryptionKey,
+                        lakekeeperDbUrlReadKey, pgConnectionString,
+                        lakekeeperDbUrlWriteKey, pgConnectionString,
+                        "LAKEKEEPER__QUEUE_CONFIG__POLL_INTERVAL", String.format("\"%s\"", task_queue_poll_interval)))
+                .withNetwork(postgresqlContainer.getNetwork())
+                .withCommand("serve").withExposedPorts(LAKEKEEPER_PORT)
+                .withExposedPorts(8181);
+        try {
+            lakekeeperCatalog.start();
+            bootstrapServer();
+            createCatalog();
+            prefix = fetchPrefix();
+        } catch (Exception e) {
+            System.err.println(lakekeeperCatalog.getLogs());
+            close();
+            throw e;
+        }
+
+    }
+
+    public void drop_without_purge(String schema, String table) {
+        Request request = Request.Builder.prepareDelete()
+                .setUri(URI.create(restUri() + "/catalog/v1/" + prefix + "/namespaces/" + schema + "/tables/" + table + "?purgeRequested=false"))
+                .setHeader("Content-Type", "application/json")
+                .build();
+        StatusResponseHandler.StatusResponse sr = HTTP_CLIENT.execute(request, createStatusResponseHandler());
+        assert sr.getStatusCode() == 204;
+    }
+
+    public String getLogs() {
+        return lakekeeperCatalog.getLogs();
+    }
```

Comment:
Inline this method.

Reply:
This method is currently used from `IcebergLakekeeperQueryRunnerMain`. In case an exception occurs, we print the logs over there. We can remove the log-dump, then we can also remove this method altogether.